bin/pycbc_make_skymap

```diff
@@ -448,6 +462,14 @@ if __name__ == '__main__':
                         action=MultiDetOptionAppendAction,
                         help='Lists of local frame files, e.g., '
                              'H1:/path/to/frame/file L1:/path/to/frame/file')
+    parser.add_argument('--injection-file', type=str,
+                        help='Optional path to an injection file.')
+    parser.add_argument('--multiifo-fake-strain', nargs='+', type=str,
+                        action=MultiDetOptionAppendAction, metavar="IFO:FAKE_STRAIN",
+                        help="The set of fake-strains for each detector")
+    parser.add_argument('--multiifo-fake-strain-seed', nargs='+', type=str,
+                        action=MultiDetOptionAppendAction, metavar="IFO:FAKE_STRAIN_SEED",
+                        help="The set of fake-strains-seeds for each detector")
```

Comment:
The "multiifo" part in the option names are not needed and may create confusion. Just `--fake-strain` and `--fake-strain-seed` should be fine (basically matching the existing options in the strain module).